projects/plugins/jetpack/tests/php/general/test_class.jetpack.php

```diff
@@ -372,52 +373,52 @@ function test_other_linked_admins_transient_set_to_zero_returns_false() {
 
 	function test_idc_optin_default() {
```

Comment:
It looks like some (possibly all) of these changed tests should be in the identity-crisis package. I don't think this is a blocker. It's fine with me if we open a new PR for moving the tests.